go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/distribution/distribution/v3 v3.0.0-20210804104954-38ab4c606ee3
 	github.com/docker/cli v20.10.8+incompatible
-	github.com/notaryproject/notation-go-lib v0.0.0-20211008041325-9b9dcd31a2ba
+	github.com/notaryproject/notation-go-lib v0.0.0-20211019185339-6f3abd71b7a0
```

Comment:
> This depends on - notaryproject/notation-go-lib#19

@sajayantony or we can merge this PR. We start another PR when notaryproject/notation-go-lib#19 is merged. In that PR, we can update the `go.mod` file and bump the version in the code to `1.0.0-alpha.1`.

Reply:
/cc @priteshbandi